src/hub.html

```diff
@@ -602,23 +602,22 @@
                 id="player-hud"
                 hud-controller="head: #player-camera;"
                 vr-mode-toggle-class="class: ui"
+                vr-mode-toggle-class__ui="class: interactable-ui"
```

Comment:
The only place I see `.interactable-ui` being used is in `media-pager`. Is that relevant here?